test/server/script-processor-test.js

```diff
@@ -320,10 +320,12 @@ describe('Script processor', () => {
         ]);
     });
 
-    it('Should expand concat operator', () => {
+    it('Should expand concat operator only if necessary', () => {
         testProcessing([
-            { src: 'prop += 1', expected: 'prop = prop + 1' },
-            { src: 'prop += 2 + prop + 1', expected: 'prop = prop + (2 + prop + 1)' }
+            { src: 'prop += 1', expected: 'prop += 1' },
+            { src: 'prop += 2 + prop + 1', expected: 'prop += 2 + prop + 1' },
```

Comment:
Why this expression is not instrumented? I thought that the problem was caused by side-effects in the **left-hand** side of the assignment. The right-hand part can be as complex as you wish.

Reply:
Otherwise, we won't handle cases like `location += '/subpath'`